modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -9,6 +9,23 @@ The default installation of metering configures Hive to use an embedded Java dat
 
 There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
 
+
+You will create your MySQL or Postgres instance with a username and password. Then either in the OpenShift CLI or by making a yaml file, you will make a secret. The `secretName` you use for this secret needs to map to your MeteringConfig `spec.hive.spec.config.db.secretName`.
+
+To create a secret in Openshift CLI you can use the following command:
+` oc --namespace <YOUR_NAMESPACE> create secret generic <YOUR_SECRETNAME> --from-literal=username=<YOUR_DATABASE_USERNAME> --from-literal=password=<YOUR_DATABASE_PASSWORD> `
```

Comment:
@lbarbeevargas @lamek is there special code formatting that ascii docs use besides backticks?

Reply:
Nevermind, looks like there's a couple of `[source,yaml]` references if I look at the raw output of this file.